react-native-libraries.json

```diff
@@ -5867,5 +5857,14 @@
     "examples": ["https://github.com/edeckers/react-native-blob-courier/tree/main/example"],
     "ios": true,
     "android": true
-  }
+  },
+  {
+    "githubUrl": "https://github.com/oblador/react-native-pinchable",
+    "examples": ["https://github.com/oblador/react-native-pinchable/tree/master/Example"],
+    "images": [
+      "https://user-images.githubusercontent.com/378279/50738295-9610d280-11d2-11e9-9dba-c0005fa9bfaf.gif"
+    ],
+    "ios": true,
+    "android": true
+  },
```

Comment:
Forgot this one. This is why I didn't add it last – to avoid conflicts on the comma :D

Reply:
Yeah, errors happens, that's why we got syntax check hooked into CI 🙂